datasets/huishoudelijkafval/ticket/v1.0.0.json

```diff
@@ -153,24 +164,52 @@
                 "description": "De reden waarom het ticket de status 'afgewezen' heeft gekregen."
               },
             "prijs": { 
-                "type": "string",
+                "type": "number",
                 "description": "De totale prijs van de gebruikte onderdelen en de uitgevoerde werkzaamheden."
             },
             "prijsOfferte": { 
-                "type": "string",
+                "type": "number",
                 "description": "De totale prijs op de offerte voor de te gebruiken onderdelen en de uit te voeren werkzaamheden."
             },
             "doorlooptijd": {
                 "type": "string",
                 "description": "De tijdsspanne in werkdagen tussen de 'datumCreatie'en de 'datumAfgerond' of de tijdsspanne in werkdagen tussen de 'datumCreatie' en de laatste statuswijziging in het geval het ticket nog niet afgerond is."
             },
             "ticketItemsDefinitief": {
```

Comment:
Maybe `ticket` in the name is redundant? Because the table that will created for this would be called `huishoudelijkafval_ticket_ticket_items_definitief` (so `ticket` is used twice in this name).

Reply:
aangepast